_sass/scale.scss

```diff
@@ -110,3 +110,22 @@ nav {
     font-size: .6em;
   }
 }
+
+/* Facebook button (news page & contact page) */
+.btn-fb {
+  font-size: 0.9rem;
+
+  /* responsive displays */
+  @media #{$mq-max-xsmall} {
+    font-size: 1rem !important;
```

Comment:
Ah, the media query **is being overwritten** as you can see in the following screenshot. I'll delete the css that overwrites it as well as the `!important` tags.

(this only happens on the news page)

<img width="333" alt="screen shot 2017-01-21 at 14 51 14" src="https://cloud.githubusercontent.com/assets/15139826/22174870/54b37882-dfe9-11e6-9acf-fba9b106fce6.png">